Library/Formula/mat.rb

```diff
@@ -0,0 +1,106 @@
+class Mat < Formula
+  desc "Metadata anonymization toolkit"
+  homepage "https://mat.boum.org/"
+  url "https://mat.boum.org/files/mat-0.5.4.tar.xz"
+  sha256 "a928cb2d5ebcafec4563b552096436771598376f8b4dded86a769c278c1314d1"
+
+  bottle do
+    cellar :any
+    sha1 "3476e92b91bddb20d386c35c997662fe08805da6" => :yosemite
+    sha1 "2f30838e07c7c342c01afa3a7faaa5cde129b4ab" => :mavericks
+    sha1 "810fe4dfe335c5dc8331734b1ea9abbe207b446d" => :mountain_lion
+  end
+
+  option "with-massive-audio", "Enable massive audio format support"
+
+  depends_on :python => :optional
+  depends_on "coreutils"
+  depends_on "poppler"
+  depends_on "intltool" => :build
+  depends_on "gettext" => :build
+
+  depends_on "exiftool" => :optional
+
+  # mat imports the "gi" python module which is not available here.
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks! :wink: